cmd/kubelet/app/server.go

```diff
@@ -108,7 +108,7 @@ type KubeletServer struct {
 	ContainerRuntime               string
 	DockerDaemonContainer          string
 	ConfigureCBR0                  bool
-	MaxPods                        int
+	Pods                        int
```

Comment:
This could remain MaxPods. Also, it looks like indentation was changed -- needs gofmt if you change it.